README.md

```diff
@@ -54,6 +68,8 @@ either run visudo and edit, or provide an alias in your .bashrc file.
 
 For example in your .bashrc set the following alias after setting up PATH:
   ```
+  export GOPATH=$HOME/gocode
```

Comment:
On a user computer $HOME/gocode most likely does not exist. I do not remember if golang creates the folder automatically while running 'go get' or it wants it to be pre-created. In the latter case, you should warn user to create the directory with mkdir